ckanext/bulk/zipoutput.py

```diff
@@ -91,7 +91,12 @@ def schema_to_csv(typ, schema_key, objects):
         header.append(field['label'].encode('utf8'))
     w.writerow(header)
     for obj in sorted(objects, key=lambda p: p['name']):
-        w.writerow([obj.get(field_name, '').encode('utf8') for field_name in field_names])
+        # fix for bpa-archive-ops#792 AttributeError: 'int' object has no attribute 'encode'
+        for field_name in field_names:
+            val = obj.get(field_name, '')
+            if isinstance(val, (int, long, float)):
+                val = str(val)
+            w.writerow(val.encode('utf8'))
```

Comment:
this looks incorrect, looks like you're going to write an entire row per field value.

it'd be better to keep the old w.writerow([...]) approach, but just wrap the obj.get() in a short function called encode_field() and put all the utf8 encoding and str casting in there.